### ISSUE TITLE
Custom Planets Broken

### ISSUE THREAD

**Issue by io-craft** [zmaster587/AdvancedRocketry#1546]
## Version of Advanced Rocketry
AdvancedRocketry-1.7.10-1.4.0-20
## Have you verified this is an issue in the latest unstable build
No.
## Version of LibVulpes
LibVulpes-1.7.10-0.2.8-34-universal.jar
## Version of Minecraft
1.7.10
## Crash report or log (if applicable)
NA.

## Description of the problem
Using custom planetDefs.xml file, with 1 planet, the server generates default planets, as well as the 1 custom planet. Even with numPlanets, etc set correctly.

**Comment by io-craft** [comment]
Seems to be somehow related to the client. Perhaps it's working (as the 1 planet i have in my custom config does work), yet maybe somehow - the Warp Controller GUI is bugged, and is adding old planets or something. Either way, its definitely buggy. Sometimes it even says "no artifact" even tho discovery is set to false etc. Can't get it to work unfortunately.

**Comment by io-craft** [comment]
GUI must be bugged somehow. I think the planets are there now, and the old ghosted ones are all gone. However I can't access the planets in the GUI now at all, but they DO appears with `/advancedRocketry planet list`:
https://i.imgur.com/1muf7X2.png

And the GUI (missing artifact error):
https://i.imgur.com/qDTUooS.png

**Comment by io-craft** [comment]
After all of that and getting them working, but the GUI still bugged, I simply just deleted the advRocketry world folder, then launched a new rocket/station #1 etc and it all came together. the mod must be doing some sort of mapping when the first station or rocket is fired.

**Comment by io-craft** [comment]
Yeah, still not working. Seems... really, really really broken. 

Removed Thermos, and even tried with just Forge. Same behavior.

**Comment by io-craft** [comment]
Resolved:

You must make sure to package both the planetDefs.xml and the advancedRocketry.cfg if you have changed the default DIM ID's at all. I have my space station DIM ID set to -20 on the server, and this needs to be included on the client or it causes a crash unrelated to above. 

In addition, if you have loaded the game with a previous planetDefs.xml or no XML at all, then you will need to purge something on the client (I would just remove the instance and redownload in my tests). Otherwise, the new planetDefs.xml will overlap with whatever the client has and its no good.

Even further, on the server, removing the `[world folder]/advRocketry/temp.dat` and replacing/removing the `[world folder]/advRocketry/planetDefs.xml` did not work. I just removed the entire `[world folder]/advRocketry` folder, Afterwards, I had to basically redo my Station 1, but it worked perfectly with existing dimensions, my station and all things.
